### PR TITLE
1.0.0 patch

### PR DESCRIPTION
**Pull request by nihgwu**

### CODE REVIEW

File: src/BaseTable.js [hunk 1]
```diff
@@ -151,9 +151,8 @@ class BaseTable extends React.PureComponent {
    */
   getTotalRowsHeight() {
     const { rowHeight, estimatedRowHeight } = this.props;
-    if (typeof estimatedRowHeight === 'number') {
```
Comment (nihgwu) [inline src/BaseTable.js:154]: let the propTypes do the check

File: src/GridTable.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ class GridTable extends React.PureComponent {
     this._setInnerRef = this._setInnerRef.bind(this);
     this._itemKey = this._itemKey.bind(this);
     this._handleItemsRendered = this._handleItemsRendered.bind(this);
-    this._getTotalRowsHeight = this._getTotalRowsHeight.bind(this);
```
Comment (nihgwu) [inline src/GridTable.js:20]: it's a method not a callback, so no need to bind the context

**Comment by nihgwu** [comment]
I don't understand but I can't push commits to your branch directly

**Comment by nihgwu** [comment]
have you granted me to edit your PR?

**Comment by nihgwu** [comment]
@jamesonhill there are still some problems to be resolved expect the column/table resize, I'll continue when I have enough time on that

**Comment by jamesonhill** [comment]
@nihgwu sorry, one moment let me see if I can give u push access. And yes you are correct, resizing still needs to be addressed. I started looking into it last night but didn't get far. Should have more time this weekend.

**Comment by jamesonhill** [comment]
@nihgwu Can you re-check you access when you have free moment? I just added you w/ "write" access.

**Comment by nihgwu** [comment]
@jamesonhill thanks, now it should work with expansion, and the performance should be better by debouncing the height change handler, and I believe there still something we can do to optimize it, like reset from a specific index instead of 0

**Pull request merged**